compiler/ballerina-lang/src/main/java/module-info.java

```diff
@@ -74,4 +74,5 @@
     exports io.ballerina.projects.plugins;
     exports io.ballerina.projects.internal.model; // TODO Remove this exports
     exports io.ballerina.projects.internal.environment; // TODO Remove these exports
+    exports io.ballerina.compiler.api.impl.symbols;
```

Comment:
Why do we expose the implementation symbols

Reply:
Because currently, I have implemented the extension with symbol API. Once [#29903](https://github.com/ballerina-platform/ballerina-lang/issues/29903) is fixed I will remove this.

Reply:
I'm not sure we can allow this. @hasithaa WDYT?

Reply:
-1, Once this is exposed, people will misuse this. Let's find a workaround.

Reply:
This is the reason for exposing.
https://github.com/ballerina-platform/ballerina-lang/pull/29920/files#r614976568
By resolving the above we can remove this

Reply:
Thanks. I removed the above export.